src/test/resources/template/ParameterReuseRecipe$1_before.java

```diff
@@ -16,7 +16,19 @@
 package foo;
 import org.openrewrite.java.*;
 
+/**
+ * OpenRewrite `before` template created for  `foo.ParameterReuseRecipe$1`.
+ */
 public class ParameterReuseRecipe$1_before {
+    /**
+     * Instantiates a new {@link ParameterReuseRecipe$1_before} instance.
+     * @return the new instance
+     */
```

Comment:
Looks like these files were not updated. (But not sure they need to be?)

Reply:
Yeah funnily enough `com.google.testing.compile:compile-testing` ignores any whitespace or comments when comparing if the outcome is the same, so I've removed the comments from the output again is it's hard to keep in sync. Thanks for pointing out the differences though! That's what prompted me to just remove those here.